Grails 7.0.0-M1 - Grails Run Command fails from IntelliJ

### Issue description

`./grailsw` does work

But Grails Run Command fails from IntelliJ does not.

![image](https://github.com/user-attachments/assets/f526b6cd-996f-4ac3-ac44-59ccb3563aa9)


```
Error: Unable to initialize main class org.grails.cli.GrailsCli
Caused by: java.lang.NoClassDefFoundError: org/gradle/tooling/BuildCancelledException
```

--- a/gradle/assemble.gradle
+++ b/gradle/assemble.gradle
@@ -140,7 +140,7 @@ class GrailsCreateStartScripts extends CreateStartScripts {
                 installedFile = "dist/$artifact.file.name"
             }
             installedFile
-        } + "lib/org.gradle/gradle-tooling-api/jars/gradle-tooling-api-${project.findProperty('gradleToolingApiVersion')}.jar".toString() // This is compileOnly, so it needs to be added manually
+        }
         generator.scriptRelPath = "bin/${unixScript.name}"
         generator.generateUnixScript(unixScript)
         generator.generateWindowsScript(windowsScript)

--- a/grails-shell/build.gradle
+++ b/grails-shell/build.gradle
@@ -4,10 +4,6 @@ mainClassName = "org.grails.cli.GrailsCli"
 
 repositories {
     mavenCentral()
-    maven {
-        url = 'https://repo.gradle.org/gradle/libs-releases'
-        description = 'Needed for Gradle Tooling API'
-    }
 }
 
 dependencies {
@@ -32,7 +28,7 @@ dependencies {
     api "org.fusesource.jansi:jansi"
     api "jline:jline"
 
-    compileOnly "org.gradle:gradle-tooling-api:$gradleToolingApiVersion"
+    api "org.gradle:gradle-tooling-api:$gradleToolingApiVersion"
     compileOnly "org.springframework.boot:spring-boot"
     compileOnly "org.springframework.boot:spring-boot-loader-tools"
     compileOnly "org.springframework:spring-web"

--- a/grails-shell/build.gradle
+++ b/grails-shell/build.gradle
@@ -27,8 +27,8 @@ dependencies {
     api "org.apache.ant:ant"
     api "org.fusesource.jansi:jansi"
     api "jline:jline"
-
     api "org.gradle:gradle-tooling-api:$gradleToolingApiVersion"
+    
     compileOnly "org.springframework.boot:spring-boot"
     compileOnly "org.springframework.boot:spring-boot-loader-tools"
     compileOnly "org.springframework:spring-web"

